test/Microsoft.AspNetCore.SystemWebAdapters.Tests/HttpBrowserCapabilitiesTests.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Web;
+using System.Web.Configuration;
+using Xunit;
+
+namespace Microsoft.AspNetCore.SystemWebAdapters.Tests
+{
+    public class HttpBrowserCapabilitiesTests
+    {
+        [Theory]
+        [InlineData("", false)]
+        [InlineData("Mozilla/5.0 (iPhone; CPU iPhone OS 12_0_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.0 Mobile/15E148 Safari/604.1", true)]
+        [InlineData("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36", false)]
+        public void IsMobileDevice(string userAgent, bool expected)
```

Comment:
There are already tests here: BrowserCapabilitiesFactoryTests. Not as discoverable, but could you put these there?

Reply:
Updated.